deployer/src/handlers/mod.rs

```diff
@@ -663,16 +663,8 @@ pub async fn get_logs(
                 .collect(),
         )),
         Err(error) => {
-            if error.code() == tonic::Code::Unavailable
-                && error.metadata().get("x-ratelimit-limit").is_some()
-            {
-                Err(Error::RateLimited(
-                    "your application is producing too many logs. Interactions with the shuttle logger service will be rate limited"
-                        .to_string(),
-                ))
-            } else {
-                Err(anyhow!("failed to retrieve logs for deployment").into())
-            }
+            error!(error = %error, "failed to retrieve logs for deployment");
+            Err(anyhow!("failed to retrieve logs for deployment").into())
```

Comment:
If this were to be reverted, it would be reverted to return a 404, but that would be [incorrect](https://github.com/shuttle-hq/shuttle/pull/1351#discussion_r1392180264), so I left this in.